packages/ui/src/components/Activity/ChannelActivitySummary.tsx

```diff
@@ -0,0 +1,173 @@
+  summary,
+  seenMarker,
+  pressHandler,
+}: {
+  summary: logic.SourceActivityEvents;
+  seenMarker: number;
+  pressHandler?: () => void;
+}) {
+  const newestPost = summary.newest;
+  const newestPostContact = useContact(newestPost.authorId ?? '');
+  const group = newestPost.group ?? undefined;
+  const channel: db.Channel | undefined = newestPost.channel ?? undefined;
+  const unreadCount =
+    summary.type === 'post'
+      ? newestPost.channel?.unread?.countWithoutThreads ?? 0
+      : newestPost.post?.threadUnread?.count ?? 0;
+
+  const newestIsBlockOrNote =
+    (summary.type === 'post' && newestPost.channel?.type === 'gallery') ||
+    newestPost.channel?.type === 'notebook';
+
+  return (
+    <View
+      padding="$l"
+      marginBottom="$l"
+      backgroundColor={
+        newestPost.timestamp > seenMarker ? '$positiveBackground' : 'unset'
+      }
+      borderRadius="$l"
+      onPress={newestIsBlockOrNote ? undefined : pressHandler}
+    >
+      <XStack>
+        <Avatar
+          contactId={newestPost.authorId ?? ''}
+          contact={newestPostContact}
+          borderRadius="$2xl"
+          size="$3xl"
+          explicitSigilSize={14}
+        />
+        <YStack marginLeft="$m">
+          {channel && (
+            <ChannelIndicator
+              unreadCount={unreadCount}
+              channel={channel}
+              group={group}
+              sentTime={newestPost.timestamp}
+            />
+          )}
+          <View>
+            <SummaryMessage summary={summary} />
+          </View>
+          <ActivitySourceContent
+            summary={summary}
+            pressHandler={pressHandler}
+          />
+        </YStack>
+      </XStack>
+    </View>
+  );
+}
+
+export function ChannelIndicator({
+  channel,
+  group,
+  unreadCount,
+  sentTime,
+}: {
+  channel: db.Channel;
+  group?: db.Group;
+  unreadCount: number;
+  sentTime?: number;
+}) {
+  const title =
+    channel.type === 'dm'
+      ? 'DM'
+      : channel.type === 'groupDm'
+        ? 'Group chat'
+        : getChannelTitle(channel);
+
+  return (
+    <XStack alignItems="center">
+      {unreadCount ? <UnreadDot marginRight="$s" /> : null}
+      <ChannelIcon channel={channel} group={group} />
+      <SizableText marginLeft="$m" fontSize="$s" color="$secondaryText">
+        {title}
+      </SizableText>
+      {sentTime && (
+        <SizableText marginLeft="$s" fontSize="$s" color="$secondaryText">
+          {logic.makePrettyTime(new Date(sentTime))}
+        </SizableText>
+      )}
+    </XStack>
+  );
+}
+
+// TODO: we dont really have a sizable channel/group icon, we use hardcoded ListItem everywhere?
+export function ChannelIcon({
+  channel,
+  group,
+}: {
+  channel: db.Channel;
+  group?: db.Group;
+}) {
+  const SIZE = 14;
+  const BORDER_RADIUS = 2;
+  const dmContact = useContact(channel.id);
+
+  if (channel.iconImage) {
+    return (
+      <View height={SIZE} width={SIZE} borderRadius={BORDER_RADIUS}>
```

Comment:
Where are you pulling design from? The one I see in Ochre uses standard sizes.

Reply:
see here:
https://www.figma.com/design/mUIcOGeELxWswuusYUnEiW/2024-Q2-Tlon.app?node-id=3026-48044&m=dev

these exact pixel values may not be correct, but our size steps definitely didn't work for rendering channel icons that tiny